pkgs/top-level/all-packages.nix

```diff
@@ -585,6 +585,7 @@ with pkgs;
   dotnet-sdk_3 = dotnetCorePackages.sdk_3_1;
   dotnet-sdk_5 = dotnetCorePackages.sdk_5_0;
   dotnet-sdk_6 = dotnetCorePackages.sdk_6_0;
+  dotnet-sdk_7 = dotnetCorePackages.sdk_7_0;
```

Comment:
I believe the runtime and aspnetcore should also be exposed in `all-packages.nix`.

Reply:
Well, I don't think it'd be a good idea to switch out the `dotnet-sdk`, `dotnet-runtime` and `dotnet-aspnetcore` packages to it because it is still a preview version, but I can add `dotnet-runtime_7` and `dotnet-aspnetcore_7` packages if that's what you mean

Reply:
We don't do that for older versions, so there is probably no reason to do that for new version.

Reply:
> Well, I don't think it'd be a good idea to switch out the dotnet-sdk, dotnet-runtime and dotnet-aspnetcore packages to it because it is still a preview version, but I can add dotnet-runtime_7 and dotnet-aspnetcore_7 packages if that's what you mean

Yeah, thats what I meant.

Reply:
Bump @GGG-KILLER

Reply:
done!